server/gokbg3/grails-app/domain/org/gokb/cred/Org.groovy

```diff
@@ -203,17 +206,19 @@ class Org extends KBComponent {
     def platforms = getProvidedPlatforms()
     def offices = getOffices()
     def identifiers = getIds()
-    
-    builder.'gokb' (attr) {
-      builder.'org' (['id':(id), 'uuid':(uuid)]) {
-       
-        addCoreGOKbXmlFields ( builder, attr )
-        builder.'homepage' (homepage)
-
-        if ( roles ) {
+
+    builder.'gokb'(attr) {
+      builder.'org'(['id': (id), 'uuid': (uuid)]) {
+
+        addCoreGOKbXmlFields(builder, attr)
+        builder.'homepage'(homepage)
+        builder.'titleNamespace'('namespaceName': titleNamespace.name, 'value': titleNamespace.value, 'id': titleNamespace.id)
+        builder.'packageNamespace'('namespaceName': packageNamespace.name, 'value': packageNamespace.value, 'id': packageNamespace.id)
```

Comment:
Needs checks for null state. Missing namespaces should result in an empty element.

Reply:
done